ircbot.py

```diff
@@ -1,10 +1,10 @@
 import sys
+import urllib.request
 
 from twisted.internet import defer, endpoints, protocol, reactor, task
 from twisted.python import log
 from twisted.words.protocols import irc
 
-
```

Comment:
shouldn't delete this whitespace, PEP0008 says two lines between imports and the rest of the code :)